MvvmCross/iOS/iOS/Views/Presenters/MvxIosViewPresenter.cs

```diff
@@ -177,15 +171,15 @@ public virtual void Show(IMvxIosView view, MvxViewModelRequest request)
                 }
             }
 
-            if (TabBarViewController != null && TabBarViewController.ShowChildView(viewController))
+            if (MasterNavigationController != null)
             {
+                PushViewControllerIntoStack(MasterNavigationController, viewController, attribute.Animated);
+
                 return;
             }
 
-            if (MasterNavigationController != null)
+            if (TabBarViewController != null && TabBarViewController.ShowChildView(viewController))
```

Comment:
Mhmmm I'm not sure about this change... we need to add a way to decide whether to prioritize a TabBarController child or a MasterNavigationController child. In any case, we should try to avoid a breaking change here, doesn't seem necessary